docker/Dockerfile

```diff
@@ -11,30 +28,41 @@ RUN apk add --no-cache --virtual .build_deps \
         libxslt-dev \
         linux-headers \
         postgresql-dev \
-        zlib-dev \
- && cd /synapse \
- && apk add --no-cache --virtual .runtime_deps \
- 	libffi \
-        libjpeg-turbo \
-	libressl \
-	libxslt \
-	libpq \
-	zlib \
-	su-exec \
- && pip install --upgrade \
+        zlib-dev
+
+RUN pip install --prefix="/install" --no-warn-script-location \
+        cryptography \
         lxml \
-        pip \
+        msgpack-python \
+        pillow \
```

Comment:
OOI, where do these 3 new entries come from? Aren't they installed as part of synapses dependencies in the next pip command? For that matter, shouldn't that also apply to those we had there already?

Reply:
So it's caching, okay :+1: